package.json

```diff
@@ -36,7 +36,7 @@
     "check-for-leaks": "^1.2.0",
     "devtron": "^1.3.0",
     "electron": "^2.0.0",
-    "electron-packager": "^8.6.0",
+    "electron-packager": "^12.0.0",
```

Comment:
You probably want to make this `^12.1.0` so the Node 10 fix is there (it was added in 12.0.2).

Reply:
Thanks @malept!